osf/models/mixins.py

```diff
@@ -1985,6 +1985,8 @@ class Meta:
     # Override on model
     SPAM_CHECK_FIELDS = {}
 
+    SPAM_CREATION_THROTTLE_LIMIT = 5
```

Comment:
This should probably go in settings, too, so that it can be easily changed by DevOps?